packages/firestore/test/unit/specs/write_spec.test.ts

```diff
@@ -595,7 +595,7 @@ describeSpec('Writes:', [], () => {
 
   specTest(
     'Writes are released when there are no queries left',
-    ['eager-gc'],
+    ['eager-gc','exclusive'],
```

Comment:
!?

Reply:
Mila must have pushed to this branch. Investigating.